### DISCUSSION TITLE
Queries related to ONEST

### DISCUSSION THREAD

**Discussion by Rajathkunder** [ONEST-Network/community#36] [Technical Queries]
Im completely new to ONEST , can anyone share me resources to go through it also which all categories are present in it , how can we onboard ourselves , domains information etc.

**Comment by mmanimegalai** [comment]
Please go through the ONEST starter pack - https://starterpack.onest.network/
In case of any specific question/s, pls add them to the same thread and we will help answer/resolve your queries
